### PR TITLE
178678 Update ffc-doc-statement-data to include all recent changes ma…

### PR DESCRIPTION
**Pull request by AbidemiAdio**
…de in ffc-pay-statement-data

### CODE REVIEW

File: docker-compose.test.debug.yaml [hunk 1]
```diff
@@ -3,4 +3,4 @@ services:
   ffc-doc-statement-data:
     command: npm run test:debug
     ports:
-      - "9252:9229"
+      - "9282:9229"
```
Comment (GodsonLeigh) [inline docker-compose.test.debug.yaml:6]: If we are changing test ports away from the default, can we update the ports doc as we go please.
https://eaflood.atlassian.net/wiki/spaces/SFI/pages/4592271438/In+use+ports+for+local+development

I will try and move this to the wiki after we have migrated the service as part of the developer setup update, but for now any port changes can be updated in the existing document so when a new service is added or updated, we don't get clashes.

**Review by GodsonLeigh — COMMENTED** [review]
comment on port usage

File: docker-compose.test.debug.yaml (hunk 1 above)
Comment (AbidemiAdio) [reply docker-compose.test.debug.yaml:6]: Many thanks, file updated.

**Review by GodsonLeigh — APPROVED** [review]
QED

**Pull request merged**